csm-extensions/services/dev-mysql/chart/values.yaml

```diff
@@ -0,0 +1,24 @@
+---
+env:
+  CF_ADMIN_PASSWORD: ~
+  CF_ADMIN_USER: ~
+  CF_CA_CERT: ~
+  CF_DOMAIN: ~
+  SERVICE_LOCATION: ~
+  SERVICE_MYSQL_HOST: "mysql"
+  SERVICE_MYSQL_PASS: "cf-usb-sidecar-mysql-admin-password"
+  SERVICE_MYSQL_PORT: 3306
+  SERVICE_MYSQL_USER: "root"
+  SERVICE_TYPE: "mysql"
+  UAA_CA_CERT: ~
```

Comment:
The variables need to get documented somewhere.

Reply:
`csm-extensions/services/dev-mysql/README.md`

Reply:
Yeah, but that's not shipped in our helm bundles.

Reply:
Ah. That was not clear from your text. I thought that you were asking for docs in the git repo.
Ok. That should be trivial to do. Simply add a reduced README into the chart directory.

Reply:
Done with commit 82d89e5e880b2d13e5d46f4086065061a5d13158